documentation/asciidoc/computers/camera/synchronous_cameras.adoc

```diff
@@ -0,0 +1,83 @@
+== Synchronous Captures
+=== IMX477
+==== Preparation
+
+Connect a potential divider of 2x 1.5KOhm Resistors to 3v3 and ground, creating 1.65V. This can be connected to either Pi.
+
+Solder the GND and XVS touchpoints of each HQ Camera board to each other.
+
+Connect the XVS wires to the 1.65V potential divider pull up.
```

Comment:
I think it'd be good to have a wiring diagram here. Perhaps something generated using [Fritzing](https://fritzing.org/).